src/Arcus.Template.WebApi/Controllers/HealthController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using GuardNet;
+
+namespace Arcus.Template.WebApi.Controllers
+{
+    /// <summary>
+    /// API endpoint to check the health of the application.
+    /// </summary>
+    [ApiController]
+    [Route("api/v1/[controller]")]
+    public class HealthController : ControllerBase
```

Comment:
Let's adhere to https://github.com/CoditEU/practical-api-guidelines/tree/master/maturity-level-one#document-your-apis

For example:

- Summary should just be `Get Health`
- OperationId `Health_Get` should be used